test/fixtures/cert-manager-values.yaml

```diff
@@ -12,4 +13,5 @@ ingressShim:
   - --v=8
   image:
     tag: build
-    pullPolicy: Never
+    repository: us.gcr.io/projet-labo-1/cert-manager-ingress-shim
+    pullPolicy: Always
```

Comment:
This needs switching back - if you do this first, I can `/test all` this 😄 (can't verify what is in that image, so I'm reluctant to test this on our infrastructure!)

Reply:
My bad, this file slipped in the commit.